ts/client/src/accounts/serum3.ts

```diff
@@ -140,6 +140,40 @@ export class Serum3Market {
     );
   }
 
+  public async computePriceForMarketOrderOfSize(
```

Comment:
is this unused?

Reply:
unused in this PR, but useful in general, max suggested to not market order but use a limit order with price which is oracle price + liquidation fee. I'd like to keep this function if ok

Reply:
ok, it also has some random constants though...